frontend/kesaseteli/youth/src/pages/already_activated.tsx

```diff
@@ -0,0 +1,21 @@
+import ActivationErrorPage from 'kesaseteli/youth/components/activation-error-page/ActivationErrorPage';
+import { GetStaticProps } from 'next';
+import { useTranslation } from 'next-i18next';
+import React from 'react';
+import getServerSideTranslations from 'shared/i18n/get-server-side-translations';
+
+const AlreadyActivatedPage: React.FC = () => {
+  const { t } = useTranslation();
+  return (
+    <ActivationErrorPage
+      title={t(`common:alreadyActivatedPage.title`)}
+      message={t(`common:alreadyActivatedPage.notificationMessage`)}
```

Comment:
in bf we use some translation bases if some translation prefixes are repeating within the component. Makes code shorter and more readable maybe. But not a blocker for the merge

Reply:
I've been thinking about it but so far I have found the full prefix more readable